imap_processing/cdf/config/imap_lo_l1a_variable_attrs.yaml

```diff
@@ -1,133 +1,182 @@
 default_attrs: &default
   # Assumed values for all variable attrs unless overwritten
   DISPLAY_TYPE: time_series
-  DEPEND_0: epoch
-  FILLVAL: -9223372036854775808
-  FORMAT: I12
-  VALIDMIN: 0
-  VALIDMAX: 9223372036854775807
-  VAR_TYPE: data
   UNITS: ' '
 
-de_default_attrs: &de_default
-  DISPLAY_TYPE: time_series
-  FILLVAL: -9223372036854775808
-  FORMAT: I12
+default_uint8_attrs: &default_uint8
+  <<: *default
+  FILLVAL: 255
+  FORMAT: I3
   VALIDMIN: 0
-  VALIDMAX: 9223372036854775807
+  VALIDMAX: 255
+  dtype: uint8
+
+default_uint16_attrs: &default_uint16
+  <<: *default
+  FILLVAL: 65535
+  FORMAT: I5
+  VALIDMIN: 0
+  VALIDMAX: 65535
+  dtype: uint16
+
+default_uint32_attrs: &default_uint32
+  <<: *default
+  FILLVAL: 4294967295
+  FORMAT: I10
+  VALIDMIN: 0
+  VALIDMAX: 4294967295
+  dtype: uint32
+
+default_uint64_attrs: &default_uint64
+  <<: *default
+  FILLVAL: 18446744073709551615
+  FORMAT: I20
+  VALIDMIN: 0
+  VALIDMAX: 18446744073709551615
+  dtype: uint64
+
+default_int32_attrs: &default_int32
+  <<: *default
+  FILLVAL: -2147483648
+  FORMAT: I10
+  VALIDMIN: -2147483648
+  VALIDMAX: 2147483647
+  dtype: int32
+
+coord_default_attrs: &coord_default
+  <<: *default
+  VAR_TYPE: support_data
+
+de_default_attrs: &de_default
+  <<: *default
   VAR_TYPE: support_data
   DEPEND_1: direct_events
-  UNITS: ' '
+
+hist_az_60_default: &hist_az_60_default
+  <<: *default_uint32
+  VAR_TYPE: data
+  DEPEND_0: epoch
+  DEPEND_1: azimuth_60
+  DEPEND_2: esa_step
+
+hist_az_6_default: &hist_az_6_default
+  <<: *default_uint32
+  VAR_TYPE: data
+  DEPEND_0: epoch
+  DEPEND_1: azimuth_6
+  DEPEND_2: esa_step
+
+spin_default_attrs: &spin_default
+  <<: *default
+  VAR_TYPE: support_data
+  DEPEND_1: spin
+
 
 # Re-used Coordinate
+epoch:
+  <<: *default_uint32
+  CATDESC: Epoch Time
+  DEPEND_0: epoch
+  FIELDNAM: Epoch time
+  Units: 'ns'
+  VAR_TYPE: support_data
+  LABLAXIS: Epoch
+
 esa_step_label:
    CATDESC: ESA Steps
    FIELDNAM: ESA Steps
    FORMAT: A1
    VAR_TYPE: metadata
 
 esa_step:
+  <<: *coord_default
+  <<: *default_uint8
```

Comment:
Oh, interesting! I hadn't thought about multiple inheritance like this. Good pattern to keep in mind.